operator/src/main/java/com/github/streamshub/console/api/v1alpha1/spec/ConsoleSpec.java

```diff
@@ -4,26 +4,37 @@
 import java.util.List;
 
 import com.fasterxml.jackson.annotation.JsonInclude;
+import com.github.streamshub.console.api.v1alpha1.spec.metrics.MetricsSource;
 
 import io.fabric8.generator.annotation.Required;
 import io.fabric8.kubernetes.api.model.EnvVar;
 import io.sundr.builder.annotations.Buildable;
 
 @Buildable(builderPackage = "io.fabric8.kubernetes.api.builder")
 @JsonInclude(JsonInclude.Include.NON_NULL)
+// Enable validation rules for unique names when array maxItems and string maxLength can be specified
+// to influence Kubernetes's estimated rule cost.
+// https://github.com/fabric8io/kubernetes-client/pull/6447
```

Comment:
Might be useful to open an issue in console to track this so we don't forget about it

Reply:
Opened https://github.com/streamshub/console/issues/1209 for this.